src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -738,10 +740,15 @@ internal override void AfterAddingTypeMembersChecks(ConversionsBase conversions,
                     diagnostics.Add(ErrorCode.ERR_AutoPropertyCannotBeRefReturning, Location);
                 }
 
-                // get-only auto property should not override settable properties
-                if (this.IsOverride && SetMethod is null && !this.IsReadOnly)
+                // Auto property should override both accessors.
+                if (this.IsOverride)
                 {
-                    diagnostics.Add(ErrorCode.ERR_AutoPropertyMustOverrideSet, Location);
+                    var overriddenProperty = (PropertySymbol)this.GetLeastOverriddenMember(this.ContainingType);
+                    if ((overriddenProperty.GetMethod is { } && GetMethod is null) ||
+                        (overriddenProperty.SetMethod is { } && SetMethod is null))
```

Comment:
Correct me if I'm wrong, but this appears to be a fundamentally different check than was here earlier. What was reporting the error on a missing `get` override earlier?

Reply:
Previously, for a set-only auto-property that overrides a virtual property with a get, the compiler would report "CS8051: Auto-implemented properties must have get accessors" but not "CS8080: Auto-implemented properties must override all accessors of the overridden property", see sharplab.io.